odxtools/comparamspec.py

```diff
@@ -21,26 +23,31 @@
 class ComparamSpec(IdentifiableElement):
     admin_data: Optional[AdminData]
     company_datas: NamedItemList[CompanyData]
+    sdgs: List[SpecialDataGroup]
     prot_stacks: NamedItemList[ProtStack]
 
     @staticmethod
-    def from_et(et_element: ElementTree.Element,
-                doc_frags: List[OdxDocFragment]) -> "ComparamSpec":
+    def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) -> "ComparamSpec":
 
         short_name = odxrequire(et_element.findtext("SHORT-NAME"))
         doc_frags = [OdxDocFragment(short_name, str(et_element.tag))]
         kwargs = dataclass_fields_asdict(IdentifiableElement.from_et(et_element, doc_frags))
 
         admin_data = AdminData.from_et(et_element.find("ADMIN-DATA"), doc_frags)
         company_datas = create_company_datas_from_et(et_element.find("COMPANY-DATAS"), doc_frags)
+        sdgs = [
+            SpecialDataGroup.from_et(sdg_element, doc_frags)
+            for sdg_element in et_element.iterfind("SDGS/SDG")
+        ]
```

Comment:
this can be done via
```suggestion
sdgs = create_sdgs_from_et(et_element.find("SDGS"), doc_frags)
```

(you have to add `from .createsdgs import create_sdgs_from_et` above.)